src/main/webapp/app/components/tokenInputGroups/TokenExtendButton.tsx

```diff
@@ -0,0 +1,178 @@
+      this.props.token,
+      moment(this.selectedDay).format(APP_LOCAL_DATETIME_FORMAT_Z_FORCE)
+    );
+  }
+
+  render() {
+    return (
+      <DefaultTooltip
+        placement={'top'}
+        overlay={'Extend token expiration date'}
+      >
+        <DropdownButton
+          as={InputGroup.Append}
+          title={<i className={classnames('fa fa-calendar')}></i>}
+          id="time-entend-dropdown"
+        >
+          <Dropdown.Item
+            eventKey="1"
+            style={{ textAlign: 'center' }}
+            onClick={() => {
+              this.extendExpirationDate(
+                this.props.token,
+                getMomentInstance(this.props.token.expiration)
+                  .add(1, 'w')
+                  .format(APP_LOCAL_DATETIME_FORMAT_Z_FORCE)
+              );
+            }}
+          >
+            {' '}
+            1 Week{' '}
+          </Dropdown.Item>
+          <Dropdown.Item
+            eventKey="2"
+            style={{ textAlign: 'center' }}
+            onClick={() => {
+              this.extendExpirationDate(
+                this.props.token,
+                getMomentInstance(this.props.token.expiration)
+                  .add(1, 'M')
+                  .format(APP_LOCAL_DATETIME_FORMAT_Z_FORCE)
+              );
+            }}
+          >
+            {' '}
+            1 Month{' '}
+          </Dropdown.Item>
+          <Dropdown.Item
+            eventKey="3"
+            style={{ textAlign: 'center' }}
+            onClick={() => {
+              this.extendExpirationDate(
+                this.props.token,
+                getMomentInstance(this.props.token.expiration)
+                  .add(6, 'M')
+                  .format(APP_LOCAL_DATETIME_FORMAT_Z_FORCE)
+              );
+            }}
+          >
+            {' '}
+            6 Months{' '}
+          </Dropdown.Item>
+          <Dropdown.Item
+            eventKey="4"
+            style={{ textAlign: 'center' }}
+            onClick={() => {
+              this.extendExpirationDate(
+                this.props.token,
+                getMomentInstance(this.props.token.expiration)
+                  .add(1, 'y')
+                  .format(APP_LOCAL_DATETIME_FORMAT_Z_FORCE)
+              );
+            }}
+          >
+            {' '}
+            1 Year{' '}
+          </Dropdown.Item>
+          <Dropdown.Divider />
+          <DayPickerInput
+            onDayChange={this.handleDayChange}
+            inputProps={{
+              style: {
+                textAlign: 'center',
+                border: 'none'
+              }
+            }}
+            style={{ margin: '0 auto' }}
+            formatDate={MomentLocaleUtils.formatDate}
+            parseDate={MomentLocaleUtils.parseDate}
+            placeholder={`Expires on: ${getMomentInstance(
+              this.props.token.expiration
+            ).format(APP_LOCAL_DATE_FORMAT)}`}
+          />
+          <Row style={{ marginTop: '2px' }}>
```

Comment:
We don't modify margin this fashion anymore. we use `mt-1` in className. Check out https://getbootstrap.com/docs/4.5/utilities/spacing/